tests/Utils/initializer.js

```diff
@@ -21,6 +21,7 @@ const LoanClosingsLiquidation = artifacts.require("LoanClosingsLiquidation");
 const LoanClosingsRollover = artifacts.require("LoanClosingsRollover");
 
 const SwapsExternal = artifacts.require("SwapsExternal");
+const SwapsImplSovrynSwapModule = artifacts.require("SwapsImplSovrynSwapModule");
```

Comment:
legacy SwapsImplSovrynSwap should be removed from setup  
making it abstract would cause tests failures

Reply:
removed in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/530/commits/0334bfa7656f28d673a27380fd05475bcc0b6524)